mobile/src/main/java/org/openhab/habdroid/ui/ChartWidgetActivity.kt

```diff
@@ -148,12 +158,7 @@ class ChartWidgetActivity : AbstractBaseActivity(), SwipeRefreshLayout.OnRefresh
     }
 
     private fun loadChartImage(force: Boolean) {
-        val connection = ConnectionFactory.activeUsableConnection?.connection
-        if (connection == null) {
-            finish()
-            return
-        }
-
+        val conn = connection ?: return
```

Comment:
```suggestion
        val conn = connection ?: return finish()
```

as in `mobile/src/main/java/org/openhab/habdroid/ui/ImageWidgetActivity.kt`

Reply:
This seems a bit redundant, since `onResume()`, which is the only place updating `connection`, already calls `finish()` if the connection is `null`. But it doesn't hurt I guess, so I added it here as well.